mission/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,7 @@
+<font color='#E3D310'>PRIMARY OBJECTIVES:</font>
+
+INSERT OBJECTIVES THAT MUST BE COMPLETED TO ACHIEVE THE MISSION GOAL
+
+<font color='#E3D310'>SECONDARY OBJECTIVES:</font>
```

Comment:
```suggestion
<font color='#E3D310'>PRIMARY OBJECTIVES</font>

INSERT OBJECTIVES THAT MUST BE COMPLETED TO ACHIEVE THE MISSION GOAL

<font color='#E3D310'>SECONDARY OBJECTIVES</font>
```

if we are keeping OBJECTIVES we should get rid of these for consistency, should also be different colours also for consistency